k2/python/tests/fsa_v2_test.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+#
+# Copyright      2021  Xiaomi Corporation      (authors: Fangjun Kuang)
+#
+# See ../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# To run this single test, use
+#
+#  ctest --verbose -R fsa_v2_test_py
+
+import unittest
+
+import k2
+import k2.ragged as k2r
+import torch
+
+
+class TestFsa(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.devices = [torch.device("cpu")]
+        if torch.cuda.is_available() and k2.with_cuda:
+            cls.devices.append(torch.device("cuda", 0))
+            if torch.cuda.device_count() > 1:
+                torch.cuda.set_device(1)
+                cls.devices.append(torch.device("cuda", 1))
+
+    def test_init_acceptor(self):
```

Comment:
@danpovey 

This shows how we access attributes of FSAs from Python, whose
underlying implementation is in C++.

Will show how to do autograd on these attributes for FSA operations in later commits.